src/Components/MovieInfo/MovieInfo.js

```diff
@@ -40,6 +40,13 @@ class MovieInfo extends React.Component {
     return (
       <main>
         <Header />
```

Comment:
Is the `Header` getting rendered in multiple components? Is it possible to pull it up to the `App` level?